rotkehlchen/data_import/importers/rotki_trades.py

```diff
@@ -34,13 +35,15 @@ def _consume_rotki_trades(
         - DivisionByZero if `amount_bought` is ZERO
         """
         fee = Fee(deserialize_asset_amount(csv_row['Fee'])) if csv_row['Fee'] else None  # noqa: E501
-        fee_currency = (
-            symbol_to_asset_or_token(csv_row['Fee Currency'])
-            if csv_row['Fee Currency'] and fee is not None else None
-        )
         amount_sold = deserialize_asset_amount(csv_row['Sell Amount'])
         amount_bought = deserialize_asset_amount(csv_row['Buy Amount'])
         location = Location.deserialize(csv_row['Location'])
+        asset_resolver = LOCATION_TO_ASSET_MAPPING.get(location, asset_from_common_identifier)
```

Comment:
same anming comments here. May also be an overkill but since the lnes of code here are identical to `rotki_events` do you think you could abstract it into a helper function?

Reply:
I think that was overkill, but I updated updated the PR with it along with the other changes. To make the function common enough I had to replace ` else Fee(ZERO)` with `else None` on [this line](https://github.com/rotki/rotki/blob/7a55f9bdb7e42e883faa4a5a8acc82a1845e7ac3/rotkehlchen/data_import/importers/rotki_events.py#L52) but `else Fee(ZERO)` didn't make sense anyway because [it's only used if it's non-zero](https://github.com/rotki/rotki/blob/7a55f9bdb7e42e883faa4a5a8acc82a1845e7ac3/rotkehlchen/data_import/importers/rotki_events.py#L72), so I changed that line too to just `if fee:`

Reply:
Great. Thanks. It looks okay now!